diagnostics-app/angular.json

```diff
@@ -0,0 +1,112 @@
+{
+  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "version": 1,
+  "newProjectRoot": "projects",
+  "projects": {
+    "diagnostics-app": {
+      "projectType": "application",
+      "schematics": {
+        "@schematics/angular:application": {
+          "strict": true
+        }
+      },
+      "root": "",
+      "sourceRoot": "src",
+      "prefix": "app",
+      "architect": {
+        "build": {
+          "builder": "@angular-devkit/build-angular:browser",
+          "options": {
+            "outputPath": "dist/diagnostics-app",
+            "index": "src/index.html",
+            "main": "src/main.ts",
+            "polyfills": "src/polyfills.ts",
```

Comment:
Do we need polyfills? Why? :)

Reply:
Yes. This file is automatically generated by Angular CLI and was earlier used to support IE and older browsers.

Although we don't need to support them, there are some ES6+ and Typescript features that might not be fully compatible yet. Hence, according to some sources, it is safe to have this file.